experimental/README.md

```diff
@@ -9,65 +9,18 @@ issues associated with it. If necessary, links are provided to additional
 documentation on an issue.  As an active Docker user and community member,
 please feel free to provide any feedback on these features you wish.
 
-## Install Docker experimental
+## Use Docker experimental
 
-Unlike the regular Docker binary, the experimental channels is built and
-updated nightly on https://experimental.docker.com. From one day to the
-next, new features may appear, while existing experimental features may be
-refined or entirely removed.
+For enabling experimental features, you need to start the Docker daemon with
+`--experimental` flag.
+If you are using systemd, you can configure the flags by using `systemctl`:
 
-1. Verify that you have `curl` installed.
+    $ sudo systemctl edit --full docker.service
 
-        $ which curl
+Then make sure the experimental flag is enabled:
```

Comment:
Perhaps we should explain how to set it through the `daemon.json` file instead? That way, we don't have to explain how to create a drop-in file for systemd, and how to `daemon-reload` etc